install.sh

```
@@ -0,0 +1,895 @@
+#!/bin/sh
+# note: we require errors to propagate (don't set -e)
+set -u
+
+PROJECT_NAME=easy-ca-cli
+OWNER=c3b2a7
+REPO="${PROJECT_NAME}"
+GITHUB_DOWNLOAD_PREFIX=https://github.com/${OWNER}/${REPO}/releases/download
+INSTALL_SH_BASE_URL=https://get.lolico.me/${PROJECT_NAME}
+LEGACY_INSTALL_SH_BASE_URL=https://raw.githubusercontent.com/${OWNER}/${PROJECT_NAME}
+PROGRAM_ARGS=("$@")
```

Shellcheck (0.10.0)

[warning] In POSIX sh, arrays are undefined.

(SC3030)